### ISSUE TITLE
Enable Nats authentication for the scanners

### ISSUE THREAD

**Issue by sleepycat** [canada-ca/tracker#3171]
Adopting Nats has worked really well for our scanners, and before we start adopting it more widely within Tracker, we should go back and sort out proper authentication between nats/scanners.

**Comment by h701h** [comment]
@FestiveKyle is this closed already?

**Comment by FestiveKyle** [comment]
Nope. This issue is for requiring credentials to connect to NATS. Each scanner, scanner processor, and the API would need the credentials in order to connect.
